CHANGELOG.md

```diff
@@ -18,10 +18,12 @@ The types of changes are:
 ## [Unreleased](https://github.com/ethyca/fides/compare/2.14.0...main)
 
 ### Added
-
+- Privacy center can now render its consent values based on Privacy Notices and Privacy Experiences [#3340](https://github.com/ethyca/fides/pull/3340)
```

Comment:
I haven't been good about updating the changelog on some of these smaller PR's... let's make sure this changelog looks ok for all the work we've put in!